volmdlr/wires.py

```diff
@@ -1337,27 +1347,6 @@ def __hash__(self):
     #         equal = (equal and prim1 == prim2)
     #     return equal
 
-    def __eq__(self, other_):
-        if other_.__class__.__name__ != self.__class__.__name__:
-            return False
-        if len(self.primitives) != len(other_.primitives):
-            return False
-        equal = 0
-        for prim1 in self.primitives:
-            reverse1 = prim1.reverse()
-            found = False
-            for prim2 in other_.primitives:
-                reverse2 = prim2.reverse()
-                if (prim1 == prim2 or reverse1 == prim2
-                        or reverse2 == prim1 or reverse1 == reverse2):
-                    equal += 1
-                    found = True
-            if not found:
-                return False
-        if equal == len(self.primitives):
-            return True
-        return False
-
```

Comment:
Do you remember what the problem with this old version was? If I am not mistaking, with your new method we don't consider the case where two contours are essentially the same, visually speaking and everything, but have primitives which are not in the same order, is that right? even in this case we consider they are different contours, then?

Reply:
![image](https://user-images.githubusercontent.com/77811165/167870541-b9684849-9311-43a1-8092-89bc66649d60.png)